<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkTask</class>
 <widget class="QWidget" name="WorkTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                stop:0 lightgreen, stop:1 darkgreen);
}
</string>
  </property>
  <widget class="QLabel" name="WorkTasklabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; font-style:italic;&quot;&gt;Work Task&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="WorkTaskTitle">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Task Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Workdate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Due Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="WorkPrority">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddTtitlWork">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: white; /* Set the background color */
    color: black;      /* Set the text color */
    border: 2px solid #56ab2f; /* Add a green border */
    border-radius: 5px; /* Optional: Rounded corners */
    padding: 5px; /* Padding for better spacing */
}
</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="AddDateWork">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: white; /* Set the background color */
    color: black;      /* Set the text color */
    border: 2px solid #56ab2f; /* Add a green border */
    border-radius: 5px; /* Optional: Rounded corners */
    padding: 5px; /* Padding for better spacing */
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AddPrioritWork">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>High </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Low</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="AddWorkButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Work Task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BacktoTaskfromWork">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to Task Manager</string>
   </property>
  </widget>
  <zorder>WorkTasklabel</zorder>
  <zorder>WorkTaskTitle</zorder>
  <zorder>Workdate</zorder>
  <zorder>WorkPrority</zorder>
  <zorder>AddTtitlWork</zorder>
  <zorder>AddPrioritWork</zorder>
  <zorder>AddWorkButton</zorder>
  <zorder>BacktoTaskfromWork</zorder>
  <zorder>AddDateWork</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
